dimagi/commcare-hq
# Sanitize mark_safe usages for reports

## Summary
Another XSS ticket as part of [SAAS-11853](https://dimagi-dev.atlassian.net/browse/SAAS-11853)

## Safety Assurance


### Automated test coverage

Small test suite in test_models.py

### QA Plan

This will need to be QA'd in conjunction with the data tables change to verify that the reports are still displaying correctly (no incorrectly-encoded HTML characters like ', ", &, <, and >)

### Safety story
Some areas were local tested where possible. The biggest concern is how this integrates with the data tables change.

### Rollback instructions
